source/facter/3.1/fact_overview.md

```diff
@@ -168,6 +197,22 @@ Facter.add(:networking, :type => :aggregate) do
 end
 ~~~
 
+The fact's output is organized by network interface into hashes, each containing the two chunks:
+
+~~~ json
```

Comment:
This isn't JSON! We should either switch the highlighter to Ruby, or fix the syntax (and quote the keys in either case).